src/Chapter2.hs

```diff
@@ -335,8 +335,13 @@ from it!
 
 ghci> :l src/Chapter2.hs
 -}
+
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList x y l
+  | x > y = []
+  | x == y = take 1 (drop x l)
+  | x < 0 || y < 0 = []
+  | otherwise = take ((y + 1) - x) (drop x l)
```

Comment:
This is absolutely correct! 👌🏼 

However, the second case is actually covered by the last last. If you would have `x == y` then the last case would be:

```haskell
| otherwise = take (x + 1 - x) (drop x l) = take 1 (drop x l)
```
See how we came to the second case 🙂